### ISSUE TITLE
How to check if consent cookie is present?

### ISSUE THREAD

**Issue by zeekrey** [Mastermindzh/react-cookie-consent#70]
Hey there, 

Once the consent is accapted and the cookie is set, it is neccessary to check the cookie on every page reload and fire the 'onAccept' function. Maybe it's a good idea to implement something to deal with this cases or at least put an example in the readme.

Thank you for your work! ✌️

**Comment by Mastermindzh** [comment]
Hey @zeekrey ,

I'm not sure I understand. Why would you have to check whether the cookie was set on every page load? And even more confusing is why you would want to call onAccept() on every page load.

The cookiebar will automatically hide once the cookie is set.

[Line 71](https://github.com/Mastermindzh/react-cookie-consent/blob/master/src/index.js#L71) makes the cookiebar visible if the cookie value is undefined (or if debug is enabled)
[Line 205](https://github.com/Mastermindzh/react-cookie-consent/blob/master/src/index.js#L205) ensures that no logic is being done (but rather a null is returned) if the bar shouldn't be visible.

Please provide me with some more details so I can try to see your point of view better :)

**Comment by zeekrey** [comment]
Hey @Mastermindzh ,

thanks for the fast reply. Sorry, for the few details. Let me try to explain it further:

Within the onAccept() function I need to add or call whatever to enable analytics or other stuff. I personally use Google Tag Manager (GTM). Thats said, I need to add the GTM script to my site. I wrapped this in a function, let's say addGTM(). Now, if the user reloads the page, this addGTM() functions needs to be called again, but only if the consent cookie is present and accepted. Currently I need to check this on my own. In addition, I need the cookie names - the default ones or the custom ones I gave them.

I was just wondering if it would be a nice idea to:

a. Add a small "how to do this" to the readme or
b. Add an exported function to this lib that can be used to check this.

The part of doing something (like addGTM()) after the consent is accepted is a somehow necessary step to make use of the whole lib.

But these are just my thoughts. 😄
